extensions/yandex-music/src/utils.ts

```diff
@@ -0,0 +1,41 @@
+import { runAppleScript } from "run-applescript";
+import { Application, getPreferenceValues, showHUD } from "@raycast/api";
+
+// May work in other browser too, but it's confirmed to work only in these
+type SupportedBrowsers = "Safari" | "Chrome" | "Yandex";
```

Comment:
I haven't seen any checks if a selected browser is different from this. Shall we show some error toast for the case user configures an unsupported browser?

Reply:
My fault. I meant that extension can work with almost every browser but i checked just these. Because actually any Chromium browser should be supported and Firefox based browsers too. The idea was just to list browsers for which it's known that extension work, not to restrict usage of extension with this list. I will correct the comment